README.md

```diff
@@ -118,10 +118,9 @@ Kaizen takes a two-pronged approach to help you find and squash bugs, both befor
 
 ### 🔧 GitHub App Setup
 
-You only need to install the GitHub app for code review. Other functionalities don't need a GitHub app.
-For PR review and description updates:
+You only need to install the GitHub app for code review (PR review and description updates). Other functionalities don't need a GitHub app.
 
-1. Follow our [GitHub App Setup Guide](https://cloudcode.ai/kaizen/docs/self_hosting_guide/setup_github_app)
+1. For the self-hosting guide, visit the [Kaizen Documentation](https://cloudcode.ai/kaizen/docs) and navigate to the "Self Hosting Guide" section.
```

Comment:
cant we pass that in the link itself?

Reply:
Here is the link: https://cloudcode.ai/kaizen/docs/self_hosting_guide/setup_github_app

Reply:
hmm seems weird, we cant load the link when it has more specific path, need to work on it to find a fix

Reply:
For now lets use what we have here, approving it

Reply:
https://www.perplexity.ai/search/why-is-it-that-i-cant-go-to-th-uHeM6WZrRkOlSkbWpRDMEA#0
Not sure why though it seems to be how github pages hosting is. 
The previous edit (https://github.com/Cloud-Code-AI/kaizen/pull/498) was this link itself.